src/app/[lang]/(protected)/chat/[chatId]/page.tsx

```diff
@@ -0,0 +1,13 @@
+import MessagesField from "./messagesField";
+import PrivateChatHeader from "./privateChatHeader";
+import MessageInput from "./messageInput";
+
+export default async function ChatID({ params }: { params: { lang: Lang } }) {
+  return (
+    <div className="mx-auto h-screen max-h-[600px] max-w-[390px] flex-col justify-between">
+      <PrivateChatHeader />
+      <MessagesField />
+      <MessageInput />
+    </div>
+  );
```

Comment:
Stretch the chat window to the full height and weight of the screen, it would look better on larger screens